### DISCUSSION TITLE
Community for discussion on jax?

### DISCUSSION THREAD

**Discussion by brando90** [jax-ml/jax#14802] [Q&A]
Pytorch has a slack channel and other communities have zulip, discord etc. 

Does jax something like that -- even if it's informal?

https://www.reddit.com/r/JAX/comments/11k80a0/community_for_discussion_on_jax/

**Comment by jakubMitura14** [comment]
Good question I was also looking for sth like that

**Comment by hawkinsp** [comment]
At the moment, we are mostly trying to foster Github discussions as the main place for community discussion. We don't have a discord or anything like that at the moment. Does Github discussions work?

**Reply by jakubMitura14** [reply]
judging on how fast You responded - yes :D

**Reply by MichaelMMeskhi** [reply]
Jax doesn't have lots of users yet and having more than 1 place will just keep things hard to communicate. I am seeing Github discussions as the best place right now.

**Comment by knightXun** [comment]
I think discord or slack is necessary for Jax Community

**Comment by neel04** [comment]
There's a [Jax LLM](https://discord.com/invite/Rf4drG3Bhp) server for some big Jax codebases (Maxtext, scalax etc.) @jakubMitura14 @knightXun 

I don't think Github discussions is sustainable in the long term.

**Comment by mdable2** [comment]
Any updates on this? Would love a JAX related Discord? Or perhaps just a discord with a channel specific to JAX stuff? But I'm finding the overlaps with category theory and cubical type theory to be very interesting. Would love to chat with more researchers on the subject!

**Reply by young-geng** [reply]
[Here's the link to the discord server.](https://discord.gg/Rf4drG3Bhp)
